sisyphus/simple_linux_utility_for_resource_management_engine.py

```diff
@@ -135,7 +135,11 @@ def options(self, rqmt):
             pass  # there is no option in SLURM?
 
         if rqmt.get('gpu', 0) > 0:
-            out.append('--gres=gpu:%s' % rqmt.get('gpu', 0))
+            gres = '--gres=gpu:'
+            if rqmt.get('gpu_name', ''):
+                gres += rqmt.get('gpu_name', '') + ':'
+            gres += str(rqmt.get('gpu', 0))
+            out.append(gres)
```

Comment:
I don't think we need defaults for the second time we call `get` with a key.
```suggestion
            gres = '--gres=gpu:'
            if rqmt.get('gpu_name', ''):
                gres += rqmt['gpu_name'] + ':'
            gres += str(rqmt['gpu'])
            out.append(gres)
```